app/models/project_media.rb

```diff
@@ -528,6 +528,19 @@ def add_nested_objects(ms)
     ms.attributes[:requests] = requests
   end
 
+  def self.create_tags(**params)
```

Comment:
In order to make the code more clear and more predictable, it's better to explicitly list all arguments that the method expects, instead of `**params` (`**params` is more common in meta-programming).